api/youtube_downloader.py

```
@@ -4,7 +4,8 @@
 import xml.etree.ElementTree as ET
 from datetime import timedelta, datetime
 import logging
-from typing import Dict, List, Optional, Union
+from typing import Dict, List, Optional, Union, Any
```

Ruff (0.11.9)

`typing.Union` imported but unused

Remove unused import: `typing.Union`

(F401)